client/ayon_core/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,162 @@
+from maya import cmds
+
+from ayon_core.hosts.maya.api import lib, plugin
+
+from ayon_core.lib import (
+    BoolDef,
+    NumberDef,
+)
+from ayon_core.pipeline import CreatedInstance
+
+
+def _get_animation_attr_defs(cls):
+    """Get Animation generic definitions."""
+    defs = lib.collect_animation_defs()
+    defs.extend(
+        [
+            BoolDef("farm", label="Submit to Farm"),
```

Comment:
Does this farm logic still work?

Reply:
Not sure, but I think that might be outside of the scope of this PR. Will make an issue to investigate.